pkg/chains/formats/slsa/internal/material/material_test.go

```diff
@@ -441,7 +443,9 @@ func TestAddSidecarImagesToMaterials(t *testing.T) {
 	}}
 	for _, tc := range tests {
 		var mat []common.ProvenanceMaterial
-		if err := AddSidecarImagesToMaterials(tc.sidecars, &mat); err != nil {
+		var err error
+		mat, err = FromSidecarImages(tc.sidecars)
+		if err != nil {
```

Comment:
In next line, comparing errors using == seems not best practice. Maybe switch to https://pkg.go.dev/errors@master#Is.

Reply:
I tried that but it doesn't work if the error is not defined as a const/var separately I think. 
Since the error can contain a unique message, I'm making sure here that the error messages are the same.